lib/coordinatorsharding.go

```diff
@@ -290,6 +292,26 @@ func (crd *Coordinator) isShardKey(bind string) bool {
 	return true
 }
 
+//Compare bind-name with scuttle ID column
+func (crd *Coordinator) isScuttleID(bindName string) bool {
+	if len(bindName) == 0 {
+		return false
+	}
+	if bindName[0] == ':' {
+		bindName = bindName[1:]
+	}
+
+	scuttleIDColumn := GetConfig().ScuttleColName
+
+	if len(bindName) < len(scuttleIDColumn) {
+		return false
+	}
+	if scuttleIDColumn == bindName {
```

Comment:
Here needs a case-insensitive comparison.

Reply:
Fixed it